app/controllers/concerns/administrate/punditize.rb

```diff
@@ -7,32 +7,58 @@ module Punditize
       included do
         private
 
+        def policy_namespaces
+          []
+        end
+
         def scoped_resource
-          policy_scope_admin super
+          namespaced_policy_scope super
```

Comment:
Another nitpick. This method relies on a private method, while the two below don't. When reading the code, this made me think how this one is special and different from the others, which I don't think it is.

I think that I would remove the private method and inline it here, and this will look this method more similar to its two siblings below, diminishing the cognitive load. Does that make sense?

```suggestion
        namespaced_scope = policy_namespaces << super
        policy_scope!(pundit_user, namespaced_scope)
```

Reply:
Totally agree. Done!